### PR TITLE
use mailto over form

### LINKED ISSUE #228: Recapcha error on /about/contact/treasurer
image

### PR DESCRIPTION
**Pull request by butlerx**
close #228

### CODE REVIEW

File: source/_posts/2017-week-23-announce.md [hunk 1]
```diff
@@ -80,8 +80,7 @@ back and if you want you can get your name on the sleeve.
 You have until the end of the month to register.
 
 Finally, if you're interested in becoming a member of the redbrick
-helpdesk team, contact us here:
-https://www.redbrick.dcu.ie/about/contact/helpdesk
+helpdesk team, contact us [here](mailto:thosecunts@redbrick.dcu.ie)
```
Comment (gruunday) [inline source/_posts/2017-week-23-announce.md]: While it's funny that it works I don't think this should be the default email for helpdesk on the site

File: source/_posts/week-6-news-events.md [hunk 2]
```diff
@@ -48,18 +48,18 @@ Our AGM will be held on the 27th of March at 6pm. Venue to be confirmed, but be
...
 
-If you want to run for one of the System Administrator positions at this year's AGM, contact [admins](http://www.redbrick.dcu.ie/about/contact/admins) to register your interest for the exam.
+If you want to run for one of the System Administrator positions at this year's AGM, contact [admins](mailto:admins@redbrick.dcu.ie) to register your interest for the exam.
 
 We will be holding the exam the week before the AGM, so please do this soon!
 
 ## Helpdesk Exam
 
-If you want to run for one of the Helpdesk positions at this year's AGM, contact [helpdesk](http://www.redbrick.dcu.ie/about/contact/helpdesk) to register your interest for the exam.
+If you want to run for one of the Helpdesk positions at this year's AGM, contact [helpdesk](mailto:thosecunts@redbrick.dcu.ie) to register your interest for the exam.
```
Comment (gruunday) [inline source/_posts/week-6-news-events.md]: Same as above

File: source/about/regulations/index.md [hunk 3]
```diff
@@ -10,7 +10,7 @@ type: about
 3. Spamming other users is an offense (e.g. mass mailing, system-wide write messages, or flooding chat rooms) unless a practical need to do so has been presented to the committee, and their permission obtained for such a mass communication.
 4. Users are requested to take all reasonable precautions to ensure the security of their account, and to comply with the instructions of Redbrick's System Administrators and with their guidelines on security.
 5. The reprovision of Dublin City University Networking Society Services or Equipment requires the approval of the committee.
-6. Certain file types are banned by our web server and users should not try to circumvent this rule. For more information on the restricted file types and getting access, contact [Helpdesk](/about/contact/helpdesk).
+6. Certain file types are banned by our web server and users should not try to circumvent this rule. For more information on the restricted file types and getting access, contact [Helpdesk](mailto:thosecunts@redbrick.dcu.ie).
```
Comment (gruunday) [inline source/about/regulations/index.md]: Same as above issues

File: source/help/index.md [hunk 4]
```diff
@@ -20,4 +20,4 @@ keywords:
 The Redbrick Helpdesk is staffed by our team of highly skilled volunteer Education Officers.
 The Helpdesk write and maintain a series of tutorials on RedBrick's systems. These are available on our wiki.
 The Helpdesk team can also be contacted by Email, and over IRC in the channel #helpdesk. You can access IRC by using one of the methods available here.
-The Helpdesk can be emailed at the [Helpdesk Email address](/about/contact/helpdesk).
+The Helpdesk can be emailed at the [Helpdesk Email address](mailto:thosecunts@redbrick.dcu.ie).
```
Comment (gruunday) [inline source/help/index.md]: Same as above issues

**Comment by butlerx** [comment]
s/thosecunts/helpdesk/g

File: source/_posts/2017-week-23-announce.md (hunk 1 above)
Comment (butlerx) [reply source/_posts/2017-week-23-announce.md]: s/thosecunts/helpdesk/g

Comment (butlerx) [inline source/_posts/2017-week-23-announce.md]: s/thosecunts/helpdesk/g

**Pull request merged**